src/components/OrganizeSidebar.tsx

```diff
@@ -1,57 +1,160 @@
-import Calendar from '@spectrum-icons/workflow/Calendar';
-import Home from '@spectrum-icons/workflow/Home';
-import Inbox from '@spectrum-icons/workflow/Inbox';
-import MapView from '@spectrum-icons/workflow/MapView';
+import grey from '@material-ui/core/colors/grey';
 import NextLink from 'next/link';
-import PeopleGroup from '@spectrum-icons/workflow/PeopleGroup';
-import User from '@spectrum-icons/workflow/User';
 import { useRouter } from 'next/router';
-import {
-    Button, Flex,
-} from '@adobe/react-spectrum';
-
+import { useState } from 'react';
+import { AppBar, Box, Button, Drawer, Hidden, IconButton, List, ListItem, Toolbar } from '@material-ui/core';
+import { Event,  Home, Inbox, Map, Menu, People, Person } from '@material-ui/icons/';
+import { makeStyles, useTheme } from '@material-ui/core/styles';
 
 interface OrganizeSidebarProps {
     orgId: string;
 }
 
+const drawerWidth = '5rem';
+
+const useStyles = makeStyles((theme) => ({
+    appBar: {
+        [theme.breakpoints.up('sm')]: {
+            display: 'none',
+        },
+    },
+    content: {
+        flexGrow: 1,
+        padding: theme.spacing(3),
+    },
+    drawer: {
+        [theme.breakpoints.up('sm')]: {
+            flexShrink: 0,
+            width: drawerWidth,
+        },
+    },
+    drawerPaper: {
+        width: drawerWidth,
+    },
+    menuButton: {
+        marginRight: theme.spacing(2),
+        [theme.breakpoints.up('sm')]: {
+            display: 'none',
+        },
+
+    },
+    root: {
+        display: 'flex',
+    },
+    roundButton: {
+        background: 'white',
+        borderRadius: '50%',
+        height: '4rem',
+    },
+    // necessary for content to be below app bar
+    toolbar: theme.mixins.toolbar,
+}));
+
 const OrganizeSidebar = ({ orgId  } : OrganizeSidebarProps) : JSX.Element =>{
+    const classes = useStyles();
+    const theme = useTheme();
+    const [mobileOpen, setMobileOpen] = useState(false);
+
+    const handleDrawerToggle = () => {
+        setMobileOpen(!mobileOpen);
+    };
+
     const router = useRouter();
     const key = router.pathname.split('[orgId]')[1];
 
+    const drawer = (
+        <Box alignItems="center" bgcolor={ grey[100] } display="flex" flexDirection="column" height="100%" justifyContent="space-between">
+            <List>
```

Comment:
On Chrome in a very wide window this is rendered a little bit too wide, causing the sidebar to overflow and a scrollbar to appear. Are you seeing the same thing?